symengine/catch/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+project(catch)
+
+set(SRC
+    catch.cpp
+)
+
+add_library(catch STATIC ${SRC})
+target_link_libraries(catch symengine)
+
+install(TARGETS catch
+            RUNTIME DESTINATION bin
+            ARCHIVE DESTINATION lib
+            LIBRARY DESTINATION lib
+        )
```

Comment:
`install` is for installing a library when `make install` is called. We don't need `catch` to be installed, right?